content/en/docs/user-journeys/users/application-developer/intermediate.md

```diff
@@ -40,15 +40,15 @@ Like Deployments, these API objects run indefinitely on a cluster until they are
   {{< note  >}}
   It's misleading to say that Deployments can't handle stateful workloads. Using {{< glossary_tooltip text="PersistentVolumes" term_id="persistent-volume" >}}, you can persist data beyond the lifecycle of any individual Pod in your Deployment.
   {{< /note  >}}
-     However, StatefulSets can provide stronger guarantees about "recovery" behavior than Deployments. StatefulSets maintain a sticky, stable identity for their Pods. The following table provides some concrete examples of what this might look like:
+  However, StatefulSets can provide stronger guarantees about "recovery" behavior than Deployments. StatefulSets maintain a sticky, stable identity for their Pods. The following table provides some concrete examples of what this might look like:
 
-    | | Deployment | StatefulSet |
-    |---|---|---|
-    | **Example Pod name** | `example-b1c4` | `example-0` |
-    | **When a Pod dies** | Reschedule on *any* node, with new name `example-a51z` | Reschedule on same node, as `example-0` |
-    | **When a node becomes unreachable** | Pod(s) are scheduled onto new node, with new names | Pod(s) are marked as "Unknown", and aren't rescheduled unless the Node object is forcefully deleted |
+⎈                                   | Deployment                                             | StatefulSet 
```

Comment:
FYI I am not 💯 about the cause of this broken table being the outer `|` symbols, because I added a similar table to https://github.com/kubernetes/website/pull/8349 and it worked with them. Also, for your empty cell in line 45, you could just try `&nbsp;` as the filler character.